csunplugged/static/js/homepage.js

```diff
@@ -0,0 +1,8 @@
+$(window).scroll(function() {
```

Comment:
It might be worth investigating if this code should be within a document ready block: https://learn.jquery.com/using-jquery-core/document-ready/

Reply:
Using the document.ready block creates a weird flash in the navbar when going from a non-homepage to the homepage.